pkg/geth/block_to_core_block_test.go

```diff
@@ -292,4 +326,58 @@ var _ = Describe("Conversion of GethBlock to core.Block", func() {
 		})
 	})
 
+	Describe("transaction error handling", func() {
+		var gethTransaction *types.Transaction
+		var gethReceipt *types.Receipt
+		var header *types.Header
+		var gethBlock *types.Block
+
+		BeforeEach(func() {
+			log.SetOutput(ioutil.Discard)
+			gethTransaction = types.NewTransaction(
```

Comment:
If the values being set in here aren't super important, I'd probably try and make them more obviously fake data so that that's clear. If they are important, maybe extract vars and assert against them.

Reply:
Makes sense. Updated.